dictcc.el

```diff
@@ -271,11 +282,23 @@ At the moment they are of the form `<tr id='trXXX'></tr>'."
                                (dictcc--insert-candidate #'cdr))))))
     (helm :prompt "Filter: " :sources (list source))))
 
+(defun dictcc--select-sourcelang-helm ()
+  "Select source language with helm."
+  (let* ((source `((name . ,"Languages" )
+		   (candidates . ,dictcc-source-languages-alist)
+		   (action . ,(quote identity))
+		   )))
+  (helm :prompt "Filter: " :sources (list source))))
+
+
 ;;;###autoload
 (defun dictcc (query)
   "Search dict.cc for QUERY and insert a result at point."
   (interactive "sQuery: \n")
-  (dictcc--request query))
+  (let ((asklang current-prefix-arg))
+    (dictcc--request query asklang))
+  )
```

Comment:
Lisp does not have trailing `)))` on their own line. The same holds further up.

Reply:
Assuming I have understood this remark as it was intended, I fixed this in all conscience.